JavaScript/JavaScriptSDK.Tests/CheckinTests/appInsights.tests.ts

```diff
@@ -1184,6 +1184,62 @@ class AppInsightsTests extends TestClass {
                 senderSpy.restore();
             }
         });
+
+        this.testCase({
+            name: "PageView should cause internal event throttle to be reset",
+            test: () => {
+                // setup
```

Comment:
Thanks for being thorough and adding tests for changes in `TelemetryContext.Track`. :+1: